chapters/descriptorsets.adoc

```diff
@@ -402,7 +402,8 @@ an <<resources-image-views, image view>> that can: be used for
 fragment shaders.
 
 All image formats that are supported for color attachments
-(ename:VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT
+(ename:VK_FORMAT_FEATURE_COLOR_ATTACHMENT_BIT,
```

Comment:
I think the redundant part here is the mention of BLEND_BIT supporting input attachment. That is already covered by COLOR_ATTACHMENT_BIT, so I think the real fix is the remove the mention of input attachment from the description of BLEND_BIT.

Reply:
Ok, I'll propose the change internally, no problem.

Reply:
You can close the PR if you wish, but the issue can stay open.